### ISSUE TITLE
Hanging in loading...

### ISSUE THREAD

**Issue by Liwellyen** [GibbsConsulting/jupyter-plotly-dash#32]
![grafik](https://user-images.githubusercontent.com/33344848/54642738-a88e6e80-4a95-11e9-90f5-db4db65230f1.png)

I tried to run the example code but there is just the loading print and nothing happens. On windows, firefox, python3.6

**Comment by GibbsConsulting** [comment]
Is this with v0.39.0 of Dash? The underlying django-plotly-dash library has an [issue](https://github.com/GibbsConsulting/django-plotly-dash/issues/127) to address the breaking changes that this version of Dash introduces. Until this is addressed, a workaround is to require v0.38.0 or earlier of Dash.

**Comment by GibbsConsulting** [comment]
A new version of django-plotly-dash has been released. @Liwellyen are you in a position to try using this version?

**Comment by GibbsConsulting** [comment]
PR #35 has been pulled into master, and released as v0.2.1
This issue should now be resolved; if not then either reopen this issue or create a new one.

**Comment by mlisovyi** [comment]
I face the same issue using on ubuntu 18.04 and py3.6.8. An example dash board in a vanilla python script works with no problem. Let me know if any further info is needed. 
```
$ conda list dash
dash                      0.43.0                   pypi_0    pypi
dash-core-components      0.43.1                   pypi_0    pypi
dash-daq                  0.1.0                    pypi_0    pypi
dash-html-components      0.13.5                   pypi_0    pypi
dash-renderer             0.24.0                   pypi_0    pypi
dash-table                3.7.0                    pypi_0    pypi
django-plotly-dash        0.9.8                    pypi_0    pypi
jupyter-plotly-dash       0.2.1                    pypi_0    pypi
```
